packages/user-interface/src/components/Menu.tsx

```diff
@@ -67,6 +71,9 @@ const Menu: FC<MenuProps> = ({ items, legacy }) => {
           const className = `denhaag-sidenav__link ${
             current && "denhaag-sidenav__link--current"
           }`;
+
+          if (!item) return <SidenavDivider key={Math.random()} />;
```

Comment:
Ik ben niet helemaal overtuigd van deze oplossing met undefined als divider en Math.random() als key, maar ik kan tegelijkertijd ook niks beters bedenken. Dus misschien is dit voor nu inderdaad wel de beste oplossing.

Reply:
Heb dit aangepast door toch de menu items in een multi array te zetten.